tests/scheduled_checks.py

```diff
@@ -3,67 +3,140 @@
-    db = Database(connection_string, reference_tables=REFERENCE_TABLES)
-    assert db
-    assert "source" in [c.name for c in db.Sources.columns]
-
-    # Load data into an in-memory sqlite database first, for performance
-
-    # create and connects to a temporary in-memory database
-    temp_db = Database("sqlite://", reference_tables=REFERENCE_TABLES)
+    db = load_astrodb(DB_NAME, data_path=DB_PATH, reference_tables=REFERENCE_TABLES)
 
-    # load the data from the data files into the database
-    temp_db.load_database(DB_PATH, verbose=False)
+    return db
 
-    # dump in-memory database to file
-    temp_db.dump_sqlite(DB_NAME)
-    # replace database object with new file version
-    db = Database("sqlite:///" + DB_NAME, reference_tables=REFERENCE_TABLES)
 
-    return db
+def test_db(db):
+    assert os.path.exists(DB_NAME)
+    assert db
+    assert "source" in [c.name for c in db.Sources.columns]
 
 
 def test_spectra_urls(db):
     spectra_urls = db.query(db.Spectra.c.access_url).astropy()
     broken_urls = []
     codes = []
-    internet, _ = internet_connection()
-    if internet:
-        for spectrum_url in tqdm(spectra_urls["access_url"]):
-            request_response = requests.head(spectrum_url)
-            status_code = request_response.status_code
-            # The website is up if the status code is 200
-            # cuny academic commons links give 301 status code
-            if status_code != 200 and status_code != 301:
-                broken_urls.append(spectrum_url)
-                codes.append(status_code)
+    # internet, _ = internet_connection()
+    # if not internet:
+    #    assert False, "No internet connection to check spectra urls"
+
+    for spectrum_url in tqdm(spectra_urls["access_url"]):
+        request_response = requests.head(spectrum_url)
+        status_code = request_response.status_code
+        # The website is up if the status code is 200
+        # cuny academic commons links give 301 status code
+        if status_code != 200 and status_code != 301:
+            broken_urls.append(spectrum_url)
+            codes.append(status_code)
 
     # Display broken spectra regardless if it's the number we expect or not
     print(f"found {len(broken_urls)} broken spectra urls: {broken_urls}, {codes}")
 
-    assert 4 <= len(broken_urls) <= 4
+    assert 5 <= len(broken_urls) <= 5
+
+# Expected fails:
+# 11123099-7653342.txt',
+# L1_OPT_2MASS_J10595138-2113082_Cruz2003.txt
+# 0000%252B2554_IRS_spectrum.fits'
+# 0415-0935.fits',
+# 2MASS+J22541892%2B3123498.fits'])
+
+
+def test_source_simbad(db):
+    # Query Simbad and confirm that there are no duplicates with different names
+
+    # Get list of all source names
+    results = db.query(db.Sources.c.source).all()
+    name_list = [s[0] for s in results]
+
+    # Add all IDS to the Simbad output as well as the user-provided id
+    Simbad.add_votable_fields("ids")
+
+    # internet_connection() is not working in the test environment
```

Comment:
I recommend just removing the internet_connection stuff instead of commenting it out